llvm/lib/Target/RISCV/AsmParser/RISCVAsmParser.cpp

```diff
@@ -3246,6 +3310,18 @@ bool RISCVAsmParser::checkPseudoAddTPRel(MCInst &Inst,
 
   return false;
 }
+bool RISCVAsmParser::checkPseudoTLSDESCCall(MCInst &Inst,
```

Comment:
Does this have a caller?

Reply:
I believe I needed to implement this for something in that tablegen created. Let me double check that though.

Reply:
Ah, that wasn't it.  I am missing the call in ProcessInstruction,  similar to the AddTPRel case.  Thanks for pointing this out.

Reply:
done